public/app/features/live/pages/types.ts

```diff
@@ -35,3 +35,16 @@ export interface GrafanaCloudBackend {
   uid: string;
   settings: any;
 }
+
+export type Setting = 'converter' | 'processor' | 'output';
```

Comment:
this name "Setting" keeps getting me confused while reading the code because "setting" is used a few different ways in this context.  Maybe RuleType?  RuleConfigType?